staging/src/kubevirt.io/client-go/examples/listvms/README.md

```diff
@@ -0,0 +1,23 @@
+# listvms
+
+List all VMs and VMIs objects in KubeVirt
+
+## How to build
+```
+$ export GO111MODULE=on
+$ go mod init
```

Comment:
There is a mod file already. Why is the `init` needed?

Reply:
These instructions are from point of view of `client-go` repo.
Someone trying to build these examples from that repo need to fetch dependencies.

Reply:
ohh got it :)

Reply:
:) Yep just building should pull them in.